core/os/device/deviceinfo/cc/BUILD.bazel

```diff
@@ -43,13 +40,10 @@ cc_library(
         "//tools/build:linux": [],
         "//tools/build:darwin": [
             "-framework Cocoa",
```

Comment:
Do we need Cocoa still?

Reply:
I guess not?

I don't have a mac to try it out. Could you have a quick go at removing this and check that it compiles?

Also, do we still need `-lgdi32` for windows?